src/wagtail_ai/static_src/image_description/main.tsx

```diff
@@ -0,0 +1,49 @@
+import './main.css';
+import { fetchResponse } from '../api';
+
+import wandIcon from '!!raw-loader!../wand_icon.svg';
```

Comment:
**Suggestion (non-blocking):** We're using Webpack 5 so we can use [asset modules](https://webpack.js.org/guides/asset-modules/) instead of `raw-loader`

Reply:
Oh, that's neat! Also, it's crazy how there's always a new and better way to do things in frontend-land 🥲